src/Editor/ImGui/OpenGLImplementation.cpp

```diff
@@ -106,7 +106,7 @@ namespace ImGuiImplementation {
             glfwGetCursorPos(g_Window, &mouse_x, &mouse_y);
             io.MousePos = ImVec2((float)mouse_x, (float)mouse_y);   // Mouse position in screen coordinates (set to -1,-1 if no mouse / on another screen, etc.)
         } else {
-            io.MousePos = ImVec2(-1,-1);
+            io.MousePos = ImVec2(-1, -1);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)